### DISCUSSION TITLE
Qt platform plugin could not be initialized - PDFlatex and/or MikTex problem

### DISCUSSION THREAD

**Discussion by lemelinm** [MiKTeX/miktex#1531] [Q&A]
Hello community,
I am using Texmaker 5.1.4 (the latest) under Windows 11 x64.  In the writing of an article, I have to use the latex command \oint (symbol of a contour integral for those who knows).  Texmaker does not have that command.  So I have to relay to use another package, namely \usepackage[intlimits]{esint}.  That the only way to use \oint.  But by doing so, I receive this error: 
![image](https://github.com/user-attachments/assets/ca533fab-676c-4f04-96e3-ffba9ef05388)
I have explored the Web to find a solution. I even installed Qt 6.7.2.  But since I am not a developer, I must say that I am stuck there and have absolutely no idea on how to make it work.
Hello community,
I am using Texmaker 5.1.4 (the latest) under Windows 11 x64.  In the writing of an article, I have to use the latex command \oint (symbol of a contour integral for those who knows).  Texmaker does not have that command.  So I have to relay to use another package, namely \usepackage[intlimits]{esint}.  That the only way to use \oint.  But by doing so, I receive this error: 
![image](https://github.com/user-attachments/assets/ca533fab-676c-4f04-96e3-ffba9ef05388)
I have explored the Web to find a solution. I even installed Qt 6.7.2.  But since I am not a developer, I must say that I am stuck there and have absolutely no idea on how to make it work.

Is there someone kind enough to help me make it work. Because I know latex but I never use any other latex software to write article and other documents. I thank you in advance for your help.
Mario

**Comment by msk-nightly** [comment]
Try the solution(s) suggested [here](https://github.com/MiKTeX/miktex/issues/1486).
